src/jazayeri_lab_to_nwb/watters/timeseries_interfaces.py

```diff
@@ -0,0 +1,207 @@
+
+        # Get processing module
+        module_description = 'Contains behavioral data from experiment.'
+        processing_module = get_module(
+            nwbfile=nwbfile, name='behavior', description=module_description)
+
+        # Add data to module
+        processing_module.add_data_interface(pupil_size)
+
+        return nwbfile
+
+
+class RewardLineInterface(TemporalAlignmentMixin):
+    """Reward line interface."""
+
+    def __init__(self, folder_path: FolderPathType):
+        # Find reward line file
+        folder_path = Path(folder_path)
+        reward_line_file = folder_path / 'reward_line.json'
+        assert reward_line_file.exists(), f'Could not find {reward_line_file}'
+        
+        # Load reward line data and set data attributes
+        reward_line_data = json.load(open(reward_line_file, 'r'))
+        self.set_original_timestamps(np.array(reward_line_data['times']))
+        self._reward_line = np.array(reward_line_data['values'])
+
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
+        # Make SpatialSeries
+        reward_line = TimeSeries(
+            name='reward_line',
+            data=H5DataIO(self._reward_line, compression='gzip'),
+            unit='reward line open',
+            timestamps=H5DataIO(self._timestamps, compression='gzip'),
+            description=(
+                'Reward line data representing events of reward dispenser'
+            ),
+        )
+
+        # Get processing module
+        module_description = 'Contains audio and reward data from experiment.'
+        processing_module = get_module(
+            nwbfile=nwbfile, name='misc', description=module_description)
+
+        # Add data to module
+        processing_module.add_data_interface(reward_line)
+
+        return nwbfile
+
+
+class AudioInterface(TemporalAlignmentMixin):
+    """Audio interface."""
+
+    def __init__(self, folder_path: FolderPathType):
+        # Find sound file
+        folder_path = Path(folder_path)
+        sound_file = folder_path / 'sound.json'
+        assert sound_file.exists(), f'Could not find {sound_file}'
+        
+        # Load sound data and set data attributes
+        sound_data = json.load(open(sound_file, 'r'))
+        self.set_original_timestamps(np.array(sound_data['times']))
+        self._audio = np.array(sound_data['values'])
+
+    def add_to_nwbfile(self, nwbfile: NWBFile, metadata: dict):
+        # Make SpatialSeries
+        audio = TimeSeries(
+            name='audio',
+            data=H5DataIO(self._audio, compression='gzip'),
+            unit='audio filename',
+            timestamps=H5DataIO(self._timestamps, compression='gzip'),
+            description='Audio data representing auditory stimuli events',
+        )
+
+        # Get processing module
+        module_description = 'Contains audio and reward data from experiment.'
+        processing_module = get_module(
+            nwbfile=nwbfile, name='misc', description=module_description)
```

Comment:
```suggestion
            nwbfile=nwbfile, name='behavior', description=module_description)
```

We usually include audio of the subject within the general 'behavior' container - is there a reason to separate it from there?